testing/endtoend/components/eth1/miner.go

```diff
@@ -211,36 +214,36 @@ func (m *Miner) Start(ctx context.Context) error {
 	m.enr = enode
 	log.Infof("Communicated enode. Enode is %s", enode)
 
-	// Connect to the started geth dev chain.
-	client, err := rpc.DialHTTP(e2e.TestParams.Eth1RPCURL(e2e.MinerComponentOffset).String())
-	if err != nil {
-		return fmt.Errorf("failed to connect to ipc: %w", err)
-	}
-	web3 := ethclient.NewClient(client)
-	block, err := web3.BlockByNumber(ctx, nil)
-	if err != nil {
-		return err
-	}
-	log.Infof("genesis block timestamp=%d", block.Time())
+	block := gen.ToBlock()
```

Comment:
the refactoring in this file is in support of this change - deriving the block from the computed geth genesis type rather than querying the EL for it. We're chucking some debugging info in the bargain, but that will be easy to bring back if we need it later.